docker/README.md

```diff
@@ -1,18 +1,28 @@
 # Get docker images
 
-You can either build the images locally, or get our "latest stable" version from the Docker hub. We recommend using the Docker hub option.
+You can either build the images locally, or get the images from the Docker hub. We recommend using the Docker hub option.
 
 ## Download images from the Docker hub
 
+**openSUSE images**
+
+    docker pull slindomansilla/openqa_data
+    docker pull slindomansilla/openqa_webui
+    docker pull slindomansilla/openqa_worker_default
```

Comment:
Waiting for feedback of the maintainer of the account [docker hub opensuse](https://hub.docker.com/u/opensuse/)